packages/expo-modules-core/ios/Swift/Exceptions/Exception.swift

```diff
@@ -25,15 +23,19 @@ open class Exception: CodedError, ChainableException, CustomStringConvertible, C
     self.origin = ExceptionOrigin(file: file, line: line, function: function)
   }
 
+  public init(name: String, description: String, file: String = #fileID, line: UInt = #line, function: String = #function) {
```

Comment:
What is the purpose of the `#` symbo here?

Reply:
That's the only way to know where exactly the exception was thrown. We use those in the `debugDescription` property which is used to print logs. See https://github.com/expo/expo/blob/ff6379d650ab0d100afe09576b233979e33be41e/packages/expo-modules-core/ios/Swift/Exceptions/Exception.swift#L40-L43

Reply:
Ohh, I see. I didn't know that swift has such functionality to get this information in this way.